executor/write.go

```diff
@@ -976,6 +977,9 @@ func (e *InsertExec) getKeysNeedCheck(rows [][]types.Datum) ([][]keyWithDupError
 	return rowWithKeys, nil
 }
 
+// filterDupRows returns rows without duplicate errors.
```

Comment:
The comment should point out that this function does BatchGet.
Or it may be misunderstood as simply remove duplicates in the input rows.